src/local-storage-lru.ts

```diff
@@ -432,13 +432,9 @@ export class LocalStorageLRU {
 
   /** Delete all keys with the given prefix */
   public deletePrefix(prefix: string): void {
-    for (let i = 0; i < this.ls.length; i++) {
-      const key = this.ls.key(i);
-      if (key == null) continue;
-      if (key.startsWith(prefix) && key !== this.recentKey) {
-        this.delete(key);
-      }
-    }
+    this.keys()
+      .filter((k) => k.startsWith(prefix) && k !== this.recentKey)
+      .forEach((k) => this.delete(k));
```

Comment:
Add try/catch as mentioned here: https://github.com/sagemathinc/local-storage-lru/issues/3#issuecomment-2189994237

Reply:
I don't want to catch the error, when deleting `recentKey`. If that fails, it's a problem with the implementation itself. The other operations are caught in the delete method and issue a warning.